### PR TITLE
Eliminate all google-analytics dead code

### PR DESCRIPTION
**Pull request by OfekShilon**

### CODE REVIEW

File: lib/csp.ts [hunk 1]
```diff
@@ -39,9 +39,6 @@ export const data = {
         'localhost:*',
```
Comment (OfekShilon) [inline lib/csp.ts:39]: All else is safe - but I'd appreciate an extra pair of eyes on the security policy file. Probably best test in staging too.

Comment (OfekShilon) [reply lib/csp.ts:39]: @AbrilRBS  does this look right to you?

Comment (mattgodbolt) [reply lib/csp.ts:39]: I'm not 100% sure we're even using this honestly (but defer to Abril 100% on this)

File: static/panes/pane.ts [hunk 2]
```diff
@@ -193,8 +178,7 @@ export abstract class Pane<S> {
     /**
      * Perform any clean-up events when the pane is closed.
      *
-     * This is typically used to emit an analytics event for closing the pane,
-     * unsubscribing from the event hub and disposing the monaco editor.
+     * This is typically used to unsubscribe from the event hub and dispose the monaco editor.
```
Comment (mattgodbolt) [inline static/panes/pane.ts:181]: Nice attention to detail; thank you

**Review by mattgodbolt — APPROVED** [review]
LGTM! Thank you :)

**Comment by partouf** [comment]
now live, looks to be working

**Pull request merged**